bindings/nodejs/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "opendal",
   "author": "OpenDAL Contributors <dev@opendal.apache.org>",
-  "version": "0.45.0",
```

Comment:
Why update this?
I don't think it should be named `x.y.z+core.a.b.c` where it relates to distribution, i.e. the version of the package to be released to the corresponding platform.
This looks so special. It's also very different from the user's daily use case.

Reply:
Let's just let the package be versioned freely. It's better to not add a core version to it.

Reply:
Make it's relation with opendal core more clear. All our packages follow the same pattern unless it's version doesn't follow semver.

Reply:
In the near future, version `0.45.3+core.0.55.0` will be released; we need to simplify how to identify the core version this package relies on.

Most importantly, users can continue to write `^0.45` without any changes on their part.

Reply:
I think we have discussed it at https://github.com/apache/opendal/discussions/4126?

Reply:
> Let's just let the package be versioned freely.

This is exactly my https://github.com/apache/opendal/discussions/4126 want to avoid.